### PR TITLE
fix(doc): Update admin configure distributed architecture

### PR DESCRIPTION
**Pull request by lpinsivy**

### CODE REVIEW

File: doc/en/administration_guide/poller/install_remote_server.rst [hunk 1]
```diff
@@ -25,12 +25,12 @@ Central Server.
 Enable Remote Server option
 ---------------------------
 
-Connect to your server and execute following command::
+Connect to your **REmoter Server** and execute following command::
```
Comment (leoncx) [inline doc/en/administration_guide/poller/install_remote_server.rst]: Some bad uppercase => REmote

Comment (lpinsivy) [reply doc/en/administration_guide/poller/install_remote_server.rst]: done

File: doc/fr/lifecycle.rst [hunk 2]
```diff
@@ -1,238 +1,124 @@
...
-+------------------------------+----------------+------------------------+
+
+=======================================
+Politique de cycle de vie des solutions
+=======================================
+
+À partir de Centreon 18.10, la société Centreon publiera de nouvelles
+versions de Centreon sur une cadence régulière, permettant à la communauté,
+aux entreprises et aux développeurs de planifier leurs feuilles de route avec
+la certitude d’avoir accès aux nouvelles sources ouvertes en amont capacités.
```
Comment (sc979) [inline doc/fr/lifecycle.rst]: I don't understand the "...ouvertes en amont capacités." part

**Comment by lpinsivy** [comment]
@sc979 @leoncx can you validate?

**Pull request merged**